openapi.yaml

```diff
@@ -1,394 +1,418 @@
-        required: true
-        schema:
-          type: string
+        - name: "id"
+          in: path
+          description: "Person ID to update"
+          required: true
+          schema:
+            type: string
       requestBody:
-        content: 
+        content:
           'application/json':
             schema:
-              $ref: '#/components/schemas/Person'
-          'application/xml':
-            schema:
-              $ref: '#/components/schemas/Person'
+              $ref: '#/components/schemas/PersonRequest'
       responses:
         "200":
           description: "Successful operation"
           content:
-            '*/*' :
+            '*/*':
               schema:
                 type: object
                 items:
-                  $ref: "#/components/schemas/Person"
-        "405":
+                  $ref: "#/components/schemas/PersonResponse"
+        "400":
           description: "Invalid input"
     delete:
       tags:
-      - "person"
-      summary: "Deletes a person"
-      description: ""
+        - "Person"
+      summary: "Delete a person"
+      description: "Delete a person with same ID"
       operationId: "deletePerson"
       parameters:
-      - name: "name"
-        in: "path"
-        description: "Person name to delete"
-        required: true
-        schema:
-          type: "string"
+        - name: "id"
+          in: "path"
+          description: "Person`s ID to delete"
+          required: true
+          schema:
+            type: "string"
       responses:
         "200":
           description: "Successful operation"
           content:
-            '*/*' :
+            '*/*':
               schema:
                 type: object
                 items:
-                  $ref: "#/components/schemas/Person"
+                  $ref: "#/components/schemas/PersonResponse"
         "400":
-            description: "Invalid name supplied"
-        "404":
-            description: "Person not found"
-  
-  /person/passport/{passportNumber}:        
-    get:
-      tags:
-      - "person"
-      summary: "Find person by passport number"
-      description: "Returns list persons"
-      operationId: "getPersonByPasswordNumber"
-      parameters:
-      - name: "passportNumber"
-        in: "path"
-        description: "number of person`s password data"
-        required: true
-        schema:
-          type: "string"
-      responses:
-        "200":
-          description: "Successful operation"
-          content:
-            '*/*' :
-              schema:
-                type: object
-                items:
-                  $ref: "#/components/schemas/Person"
-        "400":
-          description: "Invalid passport number supplied"
+          description: "Invalid ID supplied"
         "404":
-          description: "Person not found"    
-  
-  /person/passportLost/{name}:        
+          description: "Person not found"
+
+  /person/{idPerson}/passport:
     get:
```

Comment:
was updated request to return Person`s list of passports, in query we can choose statuses, active passport should be 1